chains/utils/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "utils"
```

Comment:
This name is too generic, maybe `rosetta-ethereum-testing`

Reply:
updated with `rosetta-chain-testing`